### PR TITLE
fix currently broken unit tests

### PR DESCRIPTION
**Pull request by BrandonOCasey**
added browserify debug to karma

### CODE REVIEW

File: src/html-media-source.js [hunk 1]
```diff
@@ -211,7 +216,7 @@ export default class HtmlMediaSource extends videojs.EventTarget {
     // Automatically disable the audio on the first source buffer if
     // a second source buffer is ever created
     if (this.sourceBuffers.length !== 0) {
-      this.sourceBuffers[0].disableAudio();
+      this.sourceBuffers[0].audioDisabled_ = true;
```
Comment (imbcmdth) [inline src/html-media-source.js]: Why was this changed?

File: src/html-media-source.js [hunk 2]
```diff
@@ -133,6 +133,8 @@ export default class HtmlMediaSource extends videojs.EventTarget {
     // capture the associated player when the MediaSource is
     // successfully attached
     this.on('sourceopen', (event) => {
+      // TODO: find a better way to get the player this MediaSoruce
+      // is attached to
```
Comment (imbcmdth) [inline src/html-media-source.js]: There isn't a better way unless they change MSE :smile: you might as well remove the comment.

File: src/html-media-source.js (hunk 1 above)
Comment (BrandonOCasey) [inline src/html-media-source.js]: for the test in html called "does not wrap mp4 source buffers" it was throwing an error, since native source buffers will not have a `disableAudio()` function, so I changed it to just change to property which can be created but not used on native source buffers. We may even want to change all `disableAudio` and `enableAudio` calls in html-media-source to statements like this?

**Comment by imbcmdth** [comment]
Pulling this in via #81
